usaspending_api/data/ContractSummary_download_readme.txt

```diff
@@ -0,0 +1,31 @@
+========ABOUT THESE FILES========
+
+This ZIP file was generated from a specific Contract Award Summary Page on USAspending.gov, located at https://www.usaspending.gov/#/award_v2/

```

Comment:
I believe the original comment in the file was to see if the specific ID for the award being downloaded can be used to populate these readme files with a link to the award summary page. The base URL is less-than-helpful since it resolves to a "Page not Found"

Reply:
Also, ...#/award_v2/ is the "hidden" URL, it isn't the official page and shouldn't be posted in documentation